src/js/data/names.js

```diff
@@ -3436,7 +3436,7 @@ let first = {
         ["A.C.", 5],
         ["A.J.", 63],
         ["A.W.", 64],
-        ["ATorri", 65],
```

Comment:
I jumped through `tools/names.js` and didn't see anything out of the ordinary that would have replaced it. I double checked the regex section to see if there was any possibility for false positives, but I don't see anything.

My guess is that the source data is wrong, but only you can go verify that theory!

Do you want to keep these manually-edited names, or revert it just for the sake of agreement between this file and the output of `tools/names.js`?